fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -154,6 +156,8 @@
     private static final String METER_SPLITS_TOTAL_RATIO = "queries.meter.split_queries.total_ratio";
     private static final String METER_SPLITS_RATIO = "queries.meter.split_queries.ratio";
 
+    private static final String UPTIME = "uptime";
```

Comment:
Rename this to be `"jvm.uptime"` just to be clear which uptime is being reported.

Probably rename the variable name as well